posthog/src/main/java/com/posthog/java/PostHog.java

```diff
@@ -194,4 +199,64 @@ private JSONObject getEventJson(String event, String distinctId, Map<String, Obj
         }
         return eventJson;
     }
+
+    /**
+     * 
+     * @param featureFlag which uniquely identifies your feature flag
+     *
+     * @param distinctId which uniquely identifies your user in your database. Must
+     *                   not be null or empty.
+     */
+    public boolean isFeatureFlag(String featureFlag, String distinctId) {
+        return getFeatureFlags(distinctId).get(featureFlag) != null;
```

Comment:
small bug here, the returned value can be false which is != null, which will thus return true. Add a test for this case :)

Reply:
@neilkakkar done ;)

Reply:
Hello @neilkakkar , did you have time to take a look at the last push? Thanks